endpoints/API_Player.py

```diff
@@ -6,18 +6,16 @@
 """
 from base64 import b64encode
 from .API_Interface import API_Interface
-from utils.results import Results
+from core_helpers.logging_objects import Logging_Objects
 import urllib.parse
-import logging
 
-class API_Player(Results):
+class API_Player(Logging_Objects):
     "The class that maintains the test context/state"
 
     def __init__(self, url, log_file_path=None, session_flag=False):
         "Constructor"
-
-        super(API_Player, self).__init__(
-            level=logging.DEBUG, log_file_path=log_file_path)
+        super(API_Player, self).__init__(base_url=url)
```

Comment:
We can remove `base_url` when initialising `Logging_Objects` from the `API_Player` object.

Reply:
Removed the base_url